django_fido/models.py

```diff
@@ -91,3 +101,39 @@ def attestation(self) -> AttestationObject:
     def attestation(self, value: AttestationObject):
         self.attestation_data = base64.b64encode(value).decode('utf-8')
         self.credential_id_data = base64.b64encode(value.auth_data.credential_data.credential_id).decode('utf-8')
+
+    @property
+    def key_identification(self) -> Optional[str]:
+        """Return key identification.
+
+        Depending on protocol used to register the key, this can be:
+            aaid
+            aaguid
+            attestationCertificateKeyIdentifiers
+        """
```

Comment:
Since we have multiple metadata related properties, I'd rather provide whole `AuthenticatorMetadata` object(s).

Reply:
We can add that as well. This property will be used to determine the level, so it should probably stay as a shortcut.

Reply:
We can return the whole `AuthenticatorMetadata` object instead of just its identifier.